packages/http/src/generators.ts

```diff
@@ -1,20 +1,21 @@
 import { httpFetch } from './http.common';
 import { HTTP_METHODS } from './http.constants';
+import { TalendRequestInit } from './http.types';
 
 /**
  * function - fetch a url with POST method
  *
  * @param  {string} url     url to request
  * @param  {object} payload payload to send with the request
  * @param  {object} config  option that you want apply to the request
- * @param  {object} options options to deal with cmf automatically
  * @example
  * import { http } from '@talend/http/generators';
  * import { call } from 'redux-saga/effects'
  * yield call(http.post, '/foo', {foo: 42});
  */
-export function* httpPost(url, payload, config, options) {
-	return yield* httpFetch(url, config, HTTP_METHODS.POST, payload, options);
+export function* httpPost(url: string, payload: any, config?: TalendRequestInit) {
+	// @ts-ignore
```

Comment:
why do you need this ?

Reply:
I have noooo idea of what type to put in a generator that return a yield* for an async function.
I try to dig a bit 🙄

Reply:
Actually, I don't really care as it's in generators files, I'll withdraw my change request